src/components/SpaceCard/SpaceCard.vue

```diff
@@ -69,19 +69,6 @@
           class="space-card__message"
           v-html="spaceMessage"
         />
-
-        <CardStatus
-          :opening-hours="space.openingHours"
-          class="space-card__open-status"
-          :class="{ 'space-card__open-status--visible': !hideOpeningHours }"
-        />
-
-        <OpeningHours
-          v-if="!hideOpeningHours"
```

Comment:
Remove the concept of `hideOpeningHours` from the code base. No longer need it or used.